ClientCore/TranslationNotifier.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace ClientCore.Generated;
+public static class TranslationNotifier
+{
+    public static void Register() => LiteralTranslationGenerator.Generated.TranslationNotifier.RegisterL10NCalls();
+}
```

Comment:
I am not sure why this needs to be done manually since you can generate the exact same class in the source generator itself. Same for other occurances.